ext/fetch/23_request.js

```diff
@@ -174,8 +174,8 @@ function cloneInnerRequest(request, skipBody = false) {
     body,
     redirectMode: request.redirectMode,
     redirectCount: request.redirectCount,
-    urlList: request.urlList,
-    urlListProcessed: request.urlListProcessed,
+    urlList: [request.url],
```

Comment:
Should this also be request.url()?

Reply:
I used type hints to conclude this, but to be honest I'm still grasping the meaning of the `urlList` and `urlListProcessed` by reading its use in other files. Do you understand their meaning? If not, do you know someone that implemented them and could help me?

Reply:
You're right actually, it's a list of closures. It should probably be `urlList: [() => request.url()]` to ensure that the `this` pointer is always bound correctly.

(ref)

```
/**
 * @param {(() => string)[]} urlList
 * @param {string[]} urlListProcessed
 */
function processUrlList(urlList, urlListProcessed) {
  for (let i = 0; i < urlList.length; i++) {
    if (urlListProcessed[i] === undefined) {
      urlListProcessed[i] = urlList[i]();
    }
  }
  return urlListProcessed;
}
```